pkg/fanal/analyzer/pkg/dpkg/copyright.go

```diff
@@ -130,3 +141,16 @@ func (a dpkgLicenseAnalyzer) Type() analyzer.Type {
 func (a dpkgLicenseAnalyzer) Version() int {
 	return dpkgLicenseAnalyzerVersion
 }
+
+func licensesSeparation(line string) []string {
```

Comment:
The test function is `Test_dpkgLicenseAnalyzer_splitLicenses`. We should comply with it.

```suggestion
func splitLicenses(line string) []string {
```